rubygem-gssapi/rubygem-gssapi.spec

```diff
@@ -0,0 +1,111 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name gssapi
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.2.0
+Release: 1%{?dist}
+Summary: A FFI wrapper around the system GSSAPI library
+Group: Development/Languages
+License: MIT
+URL: http://github.com/zenchild/gssapi
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix_ruby}rubygem(ffi) >= 1.0.1
```

Comment:
It looks like this package is actually from the rh-ror42 SCL rather than rh-ruby22 breaking repoclosure.

Reply:
I believe https://github.com/theforeman/foreman-packaging/pull/1835 should fix it.